sagemaker-pipeline-parameterization/notebook.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        default_value="s3://placeholder/uri/data.csv",
</pre><p> </p><p>Use SageMaker default bucket instead of "placeholder" (3 occurrences)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>